core/src/main/resources/error/error-classes.json

```diff
@@ -80,6 +80,12 @@
     ],
     "sqlState" : "42000"
   },
+  "CANNOT_PARSE_VALUE_TO_DATATYPE" : {
```

Comment:
Let's make it shorter: `INVALID_TYPED_LITERAL` taking into account that there is `UNSUPPORTED_TYPED_LITERAL` already.

Reply:
done